public/js/app.js

```diff
@@ -2013,7 +2013,43 @@ __webpack_require__.r(__webpack_exports__);
 //
```

Comment:
We shouldn't push `public/js/app.js` and `public/css/app.css` to git repo. These two are compile files (through npm) and should be put in `.gitignore`.

Reply:
ok

Reply:
This file still needs to be deleted from git. Do `git rm -f public/js/app.js` to remove it forcefully from git.